crates/subspace-runtime/src/lib.rs

```diff
@@ -185,9 +186,25 @@ pub type SS58Prefix = ConstU16<2254>;
 
 // Configure FRAME pallets to include in runtime.
 
+pub struct CallFilter;
+
+impl Contains<Call> for CallFilter {
+    fn contains(c: &Call) -> bool {
+        // Disable all balance transfers
+        !matches!(
```

Comment:
Does this handle the case of transfer call being inside of calls from `pallet-utility`?

Reply:
yes it would.